### PR TITLE
Add pagination async dropdown support

### PR DESCRIPTION
**Pull request by nearbycoder**
[sc-32499]

Add the ability to paginate a dropdown by scrolling. using https://github.com/vtaits/react-select-async-paginate/tree/master/packages/react-select-async-paginate

### CODE REVIEW

File: .github/workflows/chromatic.yml [hunk 1]
```diff
@@ -3,13 +3,6 @@
 # Workflow name
 name: 'Chromatic'
 
-# Event for the workflow
-on: 
-  push:
-    branches-ignore:
-      # Do not run on dependabot branches
-      - 'dependabot/**'
```
Comment (nearbycoder) [inline .github/workflows/chromatic.yml:11]: @hellobrian this part was to allow dependabot to run chromatic

File: lib/components/dropdown/Dropdown.jsx [hunk 2]
```diff
@@ -11,20 +12,19 @@ import { colors } from '../../theme/colors.theme';
 import { space } from '../../theme/space.theme';
 import { fontSizes, fontWeights, lineHeights } from '../../theme/font.theme';
 import { radii } from '../../theme/radii.theme';
-import { getFocusabledIn } from 'focus-lock';
 
-const getFocusStyles = state => {
+const getFocusStyles = (state) => {
   if (state.isFocused) {
     if (state.selectProps.isInvalid) {
-      return `0px 0px 0px 1px ${colors.red[70]} inset, ${colors.red[70]} 0px 0px 0px 3px`
```
Comment (nearbycoder) [inline lib/components/dropdown/Dropdown.jsx:19]: prettier changes

File: lib/components/dropdown/Dropdown.jsx [hunk 3]
```diff
@@ -215,6 +218,26 @@ export function AsyncDropdown({ ...props }) {
   );
 }
 
+export function AsyncPaginateDropdown({ ...props }) {
```
Comment (nearbycoder) [inline lib/components/dropdown/Dropdown.jsx:221]: Main add is here.

**Comment by nearbycoder** [comment]
Chromatic changes https://www.chromatic.com/build?appId=607731addb01d30021caeac2&number=475

**Review by hizegi — APPROVED** [review]
TIL about react-select-async-paginate. LGTM, but wow +26,700 for package-lock.json!

**Pull request merged**